app/assets/javascripts/application.js

```diff
@@ -20,8 +20,6 @@
 //= require leaflet-providers/leaflet-providers.js
 //= require leaflet-toolbar/dist/leaflet.toolbar.js
 //= require leaflet-distortableimage/dist/leaflet.distortableimage.js
-//= require leaflet-easybutton/src/easy-button.js
```

Comment:
@jywarren @icarito All of those I had to remove because they were not allowing me to load js and css in ``localhost:3000`